src/runners/layout.ts

```diff
@@ -0,0 +1,243 @@
+/**
+ *  Used for additionnal checks in Proof Mode
+ * when using Bitwise and Keccak builtins
+ */
+export type DilutedPool = {
+  unitsPerStep: number;
+  spacing: number;
+  nBits: number;
+};
+
+/**
+ * Layout provides a configuration when generating a proof
+ * - Builtins: List of available builtins.
+ *   segment_arena, gas_builtin and system are not proven.
```

Comment:
in doc but not in type?

Reply:
Those three builtins (`segment_arena`, `gas_builtin` and `system`) are not in the layouts because they're not proven as such (no AIR for them in the prover)

Wanted to make a note on it in docs but it is kinda unclear. I'll refactor it to a NOTE for clarity

Reply:
ah sorry my bad